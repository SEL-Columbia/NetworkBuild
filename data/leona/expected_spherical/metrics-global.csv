system (grid),system total,73.0
system (grid),system total discounted cost,10780870.3843
system (grid),system total discounted demand,22564859.9365
system (grid),system total discounted recurring cost,6430983.44084
system (grid),system total existing network length,18121.7586828
system (grid),system total external discounted recurring cost,368042.291859
system (grid),system total external initial cost,1382242.80282
system (grid),system total initial cost,4349886.94343
system (grid),system total internal discounted recurring cost,6062941.14898
system (grid),system total internal initial cost,2967644.14062
system (grid),system total levelized cost,0.477772537239
system (grid),system total proposed network length,74059.2863191
system (mini-grid),system total,29.0
system (mini-grid),system total discounted cost,2476139.83304
system (mini-grid),system total discounted demand,3446664.56801
system (mini-grid),system total discounted energy storage cost,2099988.98181
system (mini-grid),system total discounted generation cost,2196014.55549
system (mini-grid),system total discounted recurring cost,2227789.83304
system (mini-grid),system total initial cost,248350.0
system (mini-grid),system total levelized cost,0.718416249734
system (off-grid),system total,0.0
system (off-grid),system total discounted cost,0.0
system (off-grid),system total discounted demand,0.0
system (off-grid),system total discounted diesel cost,0.0
system (off-grid),system total discounted diesel fuel cost,0.0
system (off-grid),system total discounted recurring cost,0.0
system (off-grid),system total initial cost,0.0
system (off-grid),system total levelized cost,0
