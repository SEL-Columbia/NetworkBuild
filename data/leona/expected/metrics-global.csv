system (grid),system total,73.0
system (grid),system total discounted cost,10780922.3718
system (grid),system total discounted demand,22564859.9365
system (grid),system total discounted recurring cost,6430994.37256
system (grid),system total existing network length,18121.7586828
system (grid),system total external discounted recurring cost,368053.223579
system (grid),system total external initial cost,1382283.85867
system (grid),system total initial cost,4349927.99929
system (grid),system total internal discounted recurring cost,6062941.14898
system (grid),system total internal initial cost,2967644.14062
system (grid),system total levelized cost,0.477774841156
system (grid),system total proposed network length,74063.3919049
system (mini-grid),system total,29.0
system (mini-grid),system total discounted cost,2476139.83304
system (mini-grid),system total discounted demand,3446664.56801
system (mini-grid),system total discounted energy storage cost,2099988.98181
system (mini-grid),system total discounted generation cost,2196014.55549
system (mini-grid),system total discounted recurring cost,2227789.83304
system (mini-grid),system total initial cost,248350.0
system (mini-grid),system total levelized cost,0.718416249734
system (off-grid),system total,0.0
system (off-grid),system total discounted cost,0.0
system (off-grid),system total discounted demand,0.0
system (off-grid),system total discounted diesel cost,0.0
system (off-grid),system total discounted diesel fuel cost,0.0
system (off-grid),system total discounted recurring cost,0.0
system (off-grid),system total initial cost,0.0
system (off-grid),system total levelized cost,0
